docker-images/operator/scripts/launch_java.sh

```diff
@@ -12,6 +14,41 @@ function get_gc_opts {
   fi
 }
 
+# Generic formula evaluation based on awk
```

Comment:
I think the comment should explain what the formula calculates rather than stating how it calculates it.

Reply:
I've simplified the process by removing that function, which does give us much value (the goal of that function was to round a number)